allennlp/common/testing/test_case.py

```diff
@@ -24,7 +24,8 @@ class AllenNlpTestCase(TestCase):  # pylint: disable=too-many-public-methods
 
     def setUp(self):
         logging.basicConfig(format='%(asctime)s - %(levelname)s - %(name)s - %(message)s',
-                            level=logging.DEBUG)
+                            level=logging.INFO)
```

Comment:
By default, pytest shows: (1) the error, including stack traces, (2) stderr, and (3) stdout.  This means the logging statements show up at the bottom of the terminal, and because of all of our parameter and other logging when constructing models, a test involving BiDAF could have hundreds of lines of logging output that I have to scroll through before seeing the error.  It's pretty annoying.